portality/lib/formulaic.py

```diff
@@ -728,6 +728,7 @@ def __init__(self, formdata=None, source=None, parent: FormulaicContext=None):
         self._alert = []
         self._info = ''
         self._formulaic = parent
+        self._status_not_requiring_validation = ['rejected']
```

Comment:
This shouldn't be in the formulaic library file - that ought to be totally independent of the actual form configuration.

Reply:
> that ought to be totally independent of the actual form configuration.
You're right, it shouldn't be in the formulaic. It should be in the Form class - why independent of the actual form configuration? I would argue this IS the form configuration - that we may want to change in the future (ex. add other states)